packages/suite/src/views/wallet/staking/components/EthStakingDashboard/components/Transactions.tsx

```diff
@@ -1,19 +1,36 @@
 import { TransactionList } from 'src/views/wallet/transactions/TransactionList/TransactionList';
-import { useSelector } from 'src/hooks/suite';
+import { useDispatch, useSelector } from 'src/hooks/suite';
 import {
+    fetchAllTransactionsForAccountThunk,
     selectAccountStakeTypeTransactions,
     selectIsLoadingAccountTransactions,
 } from '@suite-common/wallet-core';
+import { useEffect } from 'react';
 
 export const Transactions = () => {
     const selectedAccount = useSelector(state => state.wallet.selectedAccount);
+    const accountKey = selectedAccount.account?.key || null;
+
     const transactionsIsLoading = useSelector(state =>
         selectIsLoadingAccountTransactions(state, selectedAccount.account?.key || null),
     );
     const stakeTxs = useSelector(state =>
         selectAccountStakeTypeTransactions(state, selectedAccount.account?.key || ''),
     );
 
+    const dispatch = useDispatch();
+
+    useEffect(() => {
```

Comment:
I think this should be one component up and I would not show txs when they are still loading?

Reply:
Moved it up and the loader is showing now. https://github.com/trezor/trezor-suite/pull/13788/commits/7ac22c759f5f3a938288e4984245959758b1078f